pkg/cidata/cidata.TEMPLATE.d/boot/25-guestagent-base.sh

```diff
@@ -22,6 +22,8 @@ install -m 755 "${LIMA_CIDATA_MNT}"/lima-guestagent "${LIMA_CIDATA_GUEST_INSTALL
 
 # Launch the guestagent service
 if [ -f /sbin/openrc-run ]; then
+	# Export LIMA_CIDATA_* for service
+	awk -F= '{ print $1 "=\"" $2 "\"" }' "${LIMA_CIDATA_MNT}/lima.env" >"/etc/conf.d/lima-guestagent"
```

Comment:
This doesn't work if a value contains the `=` character. I would suggest:

```suggestion
	sed -E 's/^([^=]+)=(.*)/\1="\2"/' "${LIMA_CIDATA_MNT}/lima.env" >"/etc/conf.d/lima-guestagent"
```

Please test, and add a comment _why_ this operation is necessary.